src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add() {
+      return (number, that) => {
```

Comment:
That it's not a correct name, it's better to rename it to obj 
```suggestion
      return (number, obj) => {
```